### PR TITLE
Remove redundant logs and call to save_repo_to_db

### PR DESCRIPTION
**Pull request by tapishr**
Please check the action items covered in the PR - 

- [x] Build is running
- [ ] Eventing is functional and tested
- [ ] Unit or integration tests added and running
- [x] Manual QA

### CODE REVIEW

**Review by MuskanPaliwal — APPROVED** [review]

File: vibi-dpu/src/bitbucket/repo.rs [hunk 1]
```diff
@@ -30,7 +29,6 @@ pub async fn get_workspace_repos(workspace: &str, access_token: &str) -> Option<
             None,
             "bitbucket".to_string(),
         );
-        save_repo_to_db(&val);
```
Comment (avikalpg) [inline vibi-dpu/src/bitbucket/repo.rs:33]: This used to save repos in sled db, right?
Why did you remove it?
